ui/src/main/resources/edu/wpi/grip/ui/templates/java/Pipeline.vm

```diff
@@ -0,0 +1,93 @@
+#**
+This creates the complete java pipeline.
+*##set($needsMut = false)
+#set($interpolation = false)
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.HashMap;
+
+import org.opencv.core.*;
+import org.opencv.core.Core.*;
+import org.opencv.features2d.FeatureDetector;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.*;
+
+/**
+* Pipeline class.
+*
+* <P>Autogenerated pipeline from GRIP.
```

Comment:
I think the idea is that the exporter would generate a class that has methods for providing inputs, getting outputs and running the algorithm. And that it would be a black-box kind of thing that would be instantiated in the user program. I've been away for a week and a half so I'm not sure that's still what's being generated. Do you think that that is sufficient without adding user code to the class so we wouldn't need the comment blocks?


On Jun 7, 2016, 1:18 PM -0500, Sam Carlberg<notifications@github.com>, wrote:
>  
> Inui/src/main/resources/edu/wpi/grip/ui/templates/java/Pipeline.vm(https://github.com/WPIRoboticsProjects/GRIP/pull/597#discussion_r66124695):
>  
> >+import java.util.ArrayList;>+import java.util.List;>+import java.util.Map;>+import java.util.stream.Collectors;>+import java.util.HashMap;>+>+import org.opencv.core.*;>+import org.opencv.core.Core.*;>+import org.opencv.features2d.FeatureDetector;>+import org.opencv.imgcodecs.Imgcodecs;>+import org.opencv.imgproc.*;>+>+/**>+* Pipeline class.>+*>+*<P>Autogenerated pipeline from GRIP.
>  
> Should have a warning in here that the entire file will be regenerated and any changes to it will be lost.
> Or do something like RobotBuilder and organize stuff into blocks bounded by 'Generated code' comments
>  
>  
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly,view it on GitHub(https://github.com/WPIRoboticsProjects/GRIP/pull/597/files/3c36766ceda31c3780c76a381443185fb21aa9fb#r66124695), ormute the thread(https://github.com/notifications/unsubscribe/ACM3TZVXXHrZaMPXWpfZfE6QTAMdGd7Xks5qJbXpgaJpZM4Is1zf).
>  
>  
>